vue-frontend/src/components/contact/ContactForm.vue

```diff
@@ -32,11 +33,20 @@
                 v-model="email"
                 :disabled="disableInput"
                 placeholder="Your email"
+                @input="showEmailIdValidationError = email.charAt(0) === ' '"
                 @click.native="mixpanel.track('tap_email_input')"
+                @blur="validateEmailAddress"
```

Comment:
what is use of it?

Reply:
The blur event  used to trigger an action or method when an element loses focus